.github/workflows/wheels.yml

```diff
@@ -30,13 +54,15 @@ jobs:
         # Skip building on Python 2.7 and PyPy
         # Additionally, skip 32-bit Windows for now as MSVC needs seperate setup with different toolchain to do this
         # Refer: https://cibuildwheel.readthedocs.io/en/stable/cpp_standards/#windows-and-python-27
-        CIBW_SKIP: cp27-* pp* *-win32
+        CIBW_SKIP: cp27-* pp* ${{ matrix.cibw_skip_extra }}
```

Comment:
Actually there is no need for an extra `cibw_skip_extra` variable. `*-win32` would only be present in `windows-latest` image only, it doesn't affect the others.